assets/js/modules/ads/components/common/WooCommerceRedirectModal.js

```diff
@@ -138,7 +149,10 @@ export default function WooCommerceRedirectModal( {
 	const { dismissNotification } = useDispatch( CORE_NOTIFICATIONS );
 
 	useMount( () => {
-		if ( isGoogleForWooCommerceAdsConnected ) {
+		if (
+			isWooCommerceActivated &&
+			! isAccountLinkedViaGoogleForWoocommerceNoticeDismissed
+		) {
```

Comment:
This is not how it should be done. AC says that we should dismiss that notification when the end user enters into the flow to install plugins as a result of the WooCommerce redirect modal. This means that we need to dismiss that notification when the user clicks on a CTA button, not when the modal is shown because otherwise we would dismiss it too early.

Reply:
Thanks @eugene-manuilov, this has caused me to need to raise the direction with Megan, which I will do tonight. I fully understand your comment related to the AC, however the general idea we had was that once we've communicated as such to the user, there is no need to show them a notice relating thereto again.

Let's leave this in EX with me, and I will do 2 screencasts showing Megan the scenarios in question. I will update accordingly.